bftengine/src/bftengine/ViewsManager.cpp

```diff
@@ -1107,6 +1094,9 @@ ViewChangeMsg* ViewsManager::prepareViewChangeMsgAndSetHigherView(ViewNum nextVi
     ConcordAssertNE(pVC, nullptr);
     pVC->setNewViewNumber(nextView);
     pVC->clearAllComplaints();
+    // for (auto& it : *prevViewInfo) {
```

Comment:
comments should be removed.. all 3 lines

Reply:
Done